web/yaamp/core/exchange/exchange.php

```diff
@@ -82,6 +83,8 @@ function getMarketUrl($coin, $marketName)
 		$url = "https://alcurex.com/#{$symbol}-{$base}";
 	else if($market == 'binance')
 		$url = "https://www.binance.com/trade.html?symbol={$symbol}_{$base}";
+	else if($market == 'bitfinex')
+		$url = "https://www.bitfinex.com/t/{$symbol}:{$base}";
```

Comment:
`{$lowsymbol}:{$lowbase}`

Reply:
Thank you for the review.
I will adapt accordingly.